### PR TITLE
midterm

### PR DESCRIPTION
**Pull request by JoetheJedi**
I've been checkign all sources on how to properly do the stick project but just cant seem to get it. Hoping to get some feedback on this so i can better understand how to do it

### CODE REVIEW

File: q1.py [hunk 1]
```diff
@@ -5,4 +5,6 @@
 #   Example 4: if your balance is 8, then your program should return 8. 
 
 def computeMinimumPayment( balance ):
-    #TODO write code inside this function that achieves the functionality described above
\ No newline at end of file
+ computeMinimumPayment = max(10, 0.021*balance)
```
Comment (halfaro1) [inline q1.py]: @JoetheJedi check your indentations here, remember that any portion of code that is part of a function has to be indented under that function

File: q2.py [hunk 2]
```diff
@@ -22,19 +21,8 @@ def playGame():
...
-    # 3. if there are still sticks left, return False
-    
-def determineComputerChoice():
-    # TODO: write code inside this function that returns an integer between 1 and 4, random chosen by the computer
\ No newline at end of file
+    while True:
+     UserChoice=input('How many sticks do you want to pick up? (1-4) ')
+     if UserChoice in ['1','2','3','4']:
+      break
+    print('wrong')
```
Comment (halfaro1) [inline q2.py]: @JoetheJedi good start, let's correct the indentation issues first and then we'll talk about Q2

**Comment by JoetheJedi** [comment]
Oops, i see why the first check failed, I wrote code that only worked with the example numbers. Working on it now

**Comment by JoetheJedi** [comment]
Caught myself again lol. If the balance is less than the minimum payment than the payment IS the balance.

**Comment by JoetheJedi** [comment]
4:30AM SUCCESS

**Comment by halfaro1** [comment]
@JoetheJedi 
Fantastic work :+1:
I'm marking this pull request as closed, and labeling it as `graded` while I finish grading these projects. When I'm done grading all the projects, and your grade should be on Blackboard, I'll change the tag to `grade updated on blackboard`

**Pull request closed without merging**